flink-table-store-connector/src/main/java/org/apache/flink/table/store/connector/source/FlinkSourceBuilder.java

```diff
@@ -150,20 +153,20 @@ private FileStoreSource buildFileSource(boolean isContinuous, boolean continuous
 
             LogStartupMode startupMode = conf.get(LOG_SCAN);
             if (logSourceProvider == null) {
-                return buildFileSource(true, startupMode == LogStartupMode.LATEST);
+                return buildFileSource(true, startupMode, conf.get(LOG_SCAN_TIMESTAMP_MILLS));
```

Comment:
In previous table store versions, when a table property starts with `log.`, it is meant to control the behavior of Kafka. Now you're using these properties to control both Kafka and file store.

You might need to clarify some concepts. What is a "log"? Is it only about Kafka? Shall we also consider changelog produced by file store as log?

Reply:
> Shall we also consider changelog produced by file store as log?

Yes.

We have been using `log` on incremental reading of files, such as `log.scan` for full and latest.

You can think log as changelog. It is not just for kafka.